localeval.js

```diff
@@ -1,6 +1,10 @@
 // Thaddée Tyl <thaddee.tyl@gmail.com>. License: CC-BY v3.
 "use strict";
 var node_js = typeof exports === 'object';
+if ("undefined" !== typeof process && process.browser) {
+    // webpack sets browser property on process global
+    node_js = false;
+}
```

Comment:
Also, you're using `typeof` in the commonjs section of your UMD header below to check for the existence of a global variable.